bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -123,8 +123,10 @@ spec:
                   return
             fi
             echo "Checking ${db} health"
-            serverid=$(ovsdb-tool show-log "${db}" | sed -ne '/server_id:/s/server_id: *\([[:xdigit:]]\+\)/\1/p')
-            match=$(ovsdb-tool show-log "${db}" | grep "prev_servers: *${serverid}(" || true)
+            serverid=$(ovsdb-tool show-log "${db}" | grep -m 1 -oP '(?<=server_id: )[[:xdigit:]]+')
```

Comment:
:man_shrugging: I like the original version better... I feel like it's reasonable to expect future people reading this to understand `sed` syntax but not reasonable to expect them to know GNU-specific `grep` flags... Which may be a totally unreasonable expectation?

Reply:
`grep -o` seems cleaner instead of trying to remove all the text surrounding the match.  It feels kinda sad that GNU maintainers add all these nice features (`-o` and PCRE, for example, ~20 years ago) and no one uses them.